### PR TITLE
Reset modification indicator after successfully saved changes

### PR DESCRIPTION
**Pull request by fxha**
| Q                | A
| ---------------- | ---
| Bug fix?         | yes
| License          | MIT

### Description

- Reset modification indicator after successfully saved changes
- If you want to exit with modifications and click on `Save`, the window will be closed
- Change quit message button text from `Delete` to `Exit and don't save`

@Jocs Should I open a issue for the TODO?

### CODE REVIEW

File: src/main/actions/file.js [hunk 1]
```diff
@@ -26,19 +26,31 @@ const watchAndReload = (pathname, win) => { // when i build, and failed.
...
   }
 }
 
+const writeFileToFilesystem = (pathname, content, e, callback = null) => {
+  fs.writeFile(pathname, content, 'utf-8', err => {
+    if (err) {
+      // TODO(fxha): Show user a/the error message
+      log(err)
+    } else {
+      e.sender.send('AGANI::file-saved-successfully')
```
Comment (Jocs) [inline src/main/actions/file.js]: export `PDF` and `HTML` also use the function, I think only `save markdown to file` need to send `AGANI::file-saved-successfully` ? So is it better to put `AGANI::file-saved-successfully` in the callback?

File: src/main/actions/file.js [hunk 2]
```diff
@@ -96,7 +108,7 @@ ipcMain.on('AGANI::response-close-confirm', (e, { filename, pathname, markdown }
   const win = BrowserWindow.fromWebContents(e.sender)
   dialog.showMessageBox(win, {
     type: 'warning',
-    buttons: ['Save', 'Cancel', 'Delete'],
+    buttons: ['Save', 'Cancel', 'Exit and don\'t save'],
```
Comment (Jocs) [inline src/main/actions/file.js]: I think `Delete` is the meaning of `Exit and don\'t save`, and many application use it, So I think user knew the exact meaning of `Delete`.

**Comment by Jocs** [comment]
> Should I open a issue for the TODO?

I think write in log file is enough, errors don't happen very often.

File: src/main/actions/file.js (hunk 2 above)
Comment (fxha) [reply src/main/actions/file.js]: Ok, I've never seen that before. I'll change that later.

**Comment by fxha** [comment]
@Jocs I updated changelog. Can you merge?

**Pull request merged**